system-parachains/asset-hubs/asset-hub-polkadot/src/xcm_config.rs

```diff
@@ -752,15 +768,35 @@ pub mod bridging {
 				),
 			];
 
+			pub EthereumBridgeTableV2: Vec<NetworkExportTableItem> = vec![
+				NetworkExportTableItem::new(
+					EthereumNetwork::get(),
+					Some(vec![Here]),
+					SiblingBridgeHub::get(),
+					Some((
+						XcmBridgeHubRouterFeeAssetId::get(),
+						BridgeHubEthereumBaseFeeV2::get(),
+					).into())
+				),
+			];
+
 			/// Universal aliases
 			pub UniversalAliases: BTreeSet<(Location, Junction)> = BTreeSet::from_iter(
 				vec![
+					(SiblingBridgeHubWithEthereumInboundQueueV2Instance::get(), GlobalConsensus(EthereumNetwork::get())),
 					(SiblingBridgeHubWithEthereumInboundQueueInstance::get(), GlobalConsensus(EthereumNetwork::get())),
 				]
 			);
 		}
 
-		pub type EthereumNetworkExportTable = xcm_builder::NetworkExportTable<BridgeTable>;
+		pub type EthereumNetworkExportTableV1 =
+			xcm_builder::NetworkExportTable<EthereumBridgeTableV1>;
+
+		pub type EthereumNetworkExportTableV2 =
+			snowbridge_outbound_queue_primitives::v2::XcmFilterExporter<
+				xcm_builder::NetworkExportTable<EthereumBridgeTableV2>,
+				snowbridge_outbound_queue_primitives::v2::XcmForSnowbridgeV2,
```

Comment:
I see this matches XCMs that contain `AliasOrigin` instruction - is there a way that a SnowbridgeV1 message would contain `AliasOrigin` and be wrongly routed to SnowbridgeV2?

Reply:
> is there a way that a SnowbridgeV1 message would contain AliasOrigin and be wrongly routed to SnowbridgeV2?

It's possible, but it will be trapped on BH and won't exploit the bridge, I’ve added a test for demonstration in
https://github.com/Snowfork/runtimes/pull/16